content/concepts/content-routing/overview.md

```diff
@@ -4,4 +4,56 @@ description: "Learn about Noise in libp2p."
 weight: 218
 ---
 
-Coming soon!
+## Overview
+
+Content routing refers to the process of identifying a specific peer that holds
+certain data and the means to connect to them within a P2P network.
+In a traditional client-server network, routing is a straightforward process
+because a central authority manages the flow of data. However, in a P2P network, there is
+no central authority, so routing becomes more complex as data only travels point-to-point
+(except when relayed).
+
+One of the main challenges of content routing in P2P networks is scalability. As the number
+of peers increase, it becomes more difficult to efficiently find and retrieve the desired
+content due to the larger number of potential sources.
+
+In general, specific characteristics of P2P networks complicate this process, including:
+
+- The lack of universal orchestration that a central server can provide when
+  querying and retrieving content.
+- Not having a central directory that contains information about reaching every peer
+  in the network.
+- The presence of high node churn.
+- Creating a resilient, scalable, and optimal routing protocol resistant to Sybil attacks.
+- Forward compatibility.
+
+Content routing protocols attempt to addresses these challenges by providing an effective way
+to identify a specific peer that holds the requested data.
+
+## Content Routing in libp2p
+
+libp2p provides a set of modules for different network-level functionality, including
+a content routing interface.
+
+```go
+interface ContentRouting {
+ Provide(CID, bool) error
+ FindProviders(CID) [Multiaddr]
+}
+```
+
+In libp2p, content routing is based on a
```

Comment:
It's not really based on kademlia. Kademlia is a content routing protocol that libp2p provides.

It would be more accurate to say something like:

Libp2p provides a Kademlia Implementation that applications can use to find and provide content.